### PR TITLE
feat: company-profile

### PR DESCRIPTION
**Pull request by Harmanfleet**
add terms and conditions field

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/app/pages/fleet/fuel/fuel-entry-list/fuel-entry-list.component.ts [hunk 1]
```diff
@@ -272,56 +233,68 @@ export class FuelEntryListComponent implements OnInit {
...
+          this.loaded = true;
+        }
 
+        this.suggestedUnits = [];
 
+        result[`Items`].forEach(element => {
 
-      this.fuelList = this.fuelList.concat(_.orderBy(result.Items, [(obj) => new Date(obj.data.date)], ['desc']))
+          if (element.fuelProvider == "WEX") {
+            element.dateTime = moment(element.transactionDateTime).format('MMM Do YYYY, h:mm a')
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/fuel/fuel-entry-list/fuel-entry-list.component.ts]: What does the `Do` signify in `MMM Do YYYY, h:mm a`

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]
@Harmanfleet please comment on the question

**Comment by kunalfleethawks** [comment]
please resolve the conflict

File: src/app/pages/fleet/fuel/fuel-entry-list/fuel-entry-list.component.html [hunk 2]
```diff
@@ -61,97 +61,139 @@
...
-                    <ngx-datatable-column name="Fuel Type" prop="data.type">
-                    </ngx-datatable-column>
-                    <ngx-datatable-column name="Fuel Amount" prop="data.amt">
-                    </ngx-datatable-column>
-                    <ngx-datatable-column name="Fuel Units" prop="data.qty">
-                    </ngx-datatable-column>
-                    <ngx-datatable-column name="Site" prop="data.site">
-                    </ngx-datatable-column>
-                    <ngx-datatable-column name="Province" prop="data" [width]=300 [sortable]=false>
-                      <ng-template ngx-datatable-cell-template let-rowIndex="rowIndex" let-value="value"
```
Comment (FH-sandeepkaur1) [inline src/app/pages/fleet/fuel/fuel-entry-list/fuel-entry-list.component.html]: Please rectify this code

File: src/app/pages/manage/company/company-profile/company-profile.component.html [hunk 3]
```diff
@@ -224,13 +225,72 @@ <h2 class="font-weight-normal text-dark text-5 mt-0 mb-1">Bank Address</h2>
...
+                        <td width="30%">{{carriers.termsCarrier}}</td>
+                      </tr>
+                      <tr class="text-capitalize">
+                        <td class="back-color-gray">Terms and Conditions For Truck Dispatch</td>
+                        <td>{{carriers.notesFactor}}</td>
+                        <td class="back-color-gray">Notes For B.O.L</td>
+                        <td>{{carriers.notesBol}}</td>
+                      </tr>
+                      <tr class="text-capitalize">
+                        <td class="back-color-gray">Dispatch Notes For Driver</td>
```
Comment (FH-sandeepkaur1) [inline src/app/pages/manage/company/company-profile/company-profile.component.html:286]: Please check your binding variables

**Review by FH-sandeepkaur1 — CHANGES_REQUESTED** [review]

**Review by FH-sandeepkaur1 — APPROVED** [review]

**Review by kunalfleethawks — APPROVED** [review]

**Pull request merged**